contracts/governance/DESIGN.md

```diff
@@ -0,0 +1,59 @@
+## Axelar Governance Protocol
+
+# Multisig
+Central to the Axelar Governance Protocol is a Multisig contract, incorporating MultisigBase and Multisig components. This contract maintains a list of signer accounts and enforces a threshold or minimum number of signers required to execute certain functions. It plays a pivotal role in defining token mint limits for the gateway.
```

Comment:
This isn't central anymore. Main contract relies on GMP via timelocks. Multisig is only relevant for the token mint limit feature now